contents/tutorials/android-ab-tests.md

```diff
@@ -0,0 +1,273 @@
+    Column(modifier = modifier.fillMaxSize()) {
+        Text(text = "Hello $name!")
+        Button(onClick = { navController.navigate("second_screen") }) {
+            Text("Go to next screen")
+        }
+    }
+}
+
+@Composable
+fun SecondScreen() {
+    val isTestVariant = remember { mutableStateOf(false) } // We'll set this value later in the tutorial
+
+    val backgroundColor = if (isTestVariant.value) Color.Green else Color.Red
+
+    Surface(modifier = Modifier.fillMaxSize(), color = backgroundColor) {
+
+        Column(modifier = Modifier.fillMaxSize(), horizontalAlignment = Alignment.CenterHorizontally, verticalArrangement = Arrangement.Center) {
+            Button(onClick = { /* Do nothing for now */ }) {
+                Text("Click Me!")
+            }
+        }
+    }
+}
+
+@Preview(showBackground = true)
+@Composable
+fun DefaultPreview() {
+    MyABTestTheme {
+        MyApp()
+    }
+}
+```
+
+Make sure to add `implementation("androidx.navigation:navigation-compose:2.4.0")` to your dependencies in to `build.gradle (Module: app)`.
+
+Our basic set up is now complete. Build and run your app to test that it's working.
+
+![Basic setup of the Android app](../images/tutorials/android-ab-tests/android-basic-setup.mp4)
+
+## Adding PostHog to your Android app
+
+First, add the [PostHog Android SDK](/docs/libraries/android) as a dependency in your `build.gradle` file. You can find the latest version on our [GitHub](https://github.com/PostHog/posthog-android/blob/main/CHANGELOG.md). For this tutorial, we use version `2.0.3`.
+
+
+```gradle
+dependencies {
+    implementation("com.posthog.android:posthog:2.0.3")
+}
+```
+
+Sync your project with your Gradle file changes.
+
+Next, we create a custom application class where we can configure our PostHog instance. In a new file `MyABTestApplication.kt`, add the following code:
+
+```kotlin
+// in MyABTestApplication.kt
+package com.example.my_ab_test
+
+import android.app.Application
+import com.posthog.android.PostHog
+
+class MyABTestApplication : Application() {
+    companion object {
+        private const val POSTHOG_API_KEY = "<ph_project_api_key>"
+        private const val POSTHOG_HOST = "<ph_instance_address>"
+    }
+
+    override fun onCreate() {
+        super.onCreate()
+
+        // Initialize PostHog
+        val posthog = PostHog.Builder(this, POSTHOG_API_KEY, POSTHOG_HOST)
+            .captureApplicationLifecycleEvents()
+            .recordScreenViews()
+            .build()
+
+        // Set the initialized instance as a globally accessible instance
+        PostHog.setSingletonInstance(posthog)
+    }
+}
+```
+
+To get your PostHog API key and host, [sign up to PostHog](https://app.posthog.com/signup). Then, you can find your API key and host in your [project settings](https://app.posthog.com/settings/project). Replace the strings `<ph_project_api_key>` and `<ph_instance_address>` with your API key and host.
```

Comment:
ph_project_api_key and ph_instance_address are automatically replaced in code so you don't need to mention it, and it might be confusing.
```suggestion
To get your PostHog API key and host, [sign up to PostHog](https://app.posthog.com/signup). Then, you can find your API key and host in your [project settings](https://app.posthog.com/settings/project).
```